src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -138,20 +140,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
+    String type = request.getParameter("type");
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    if(type != null){
+      if(!conversation.getType().equals(type)){
+        conversation.makePrivate();
+        conversationStore.updateConversation(conversation);
+        System.out.println("Done!");
```

Comment:
If the purpose of the log is debugging, maybe the message should be a little clearer? Something like "Conversation <convo_name> switched to private by <user_id>"?

Reply:
Oops, I had forgotten to remove that one.